devel/onetbb/Portfile

```diff
@@ -34,4 +34,6 @@ depends_lib-append  port:hwloc
 
 compiler.cxx_standard  2011
 
-configure.args-append  -DTBB_TEST=OFF
+cmake.set_osx_architectures no
```

Comment:
very unusual

Reply:
@kencu it requires some time to dig

Reply:
@catap Allow me to test this for `ppc32` with my patches.

Reply:
@kencu  / @barracuda156 just added explanation, because it really isn't obviously

Reply:
@barracuda156 I assume that you may need to adjust your ppc32 patch.